corehq/motech/value_source.py

```diff
@@ -246,11 +246,13 @@ def _recurse_form_questions(form_dict, path, result_):
     # familiar from form builder. The form's data will usually be immediately under "form_json['form']" but not
     # necessarily. If this causes problems we may need a more reliable way to get to it.
 
-    if 'meta' in form_json['form'] and 'received_on' in form_json:
-        metadata = {
-            'timeStart': form_json['form']['meta']['timeStart'],
-            'timeEnd': form_json['form']['meta']['timeEnd'],
-            'received_on': form_json['received_on'],
-        }
```

Comment:
i was seeing this more like
```
if 'meta' in form_json['form']:
    if 'timeStart' in form_json['form']['meta']:
        metadata['timeStart'] = form_json['form']['meta']['timeStart']
    # ... similarly for timeEnd
```
just a little easier to read.

Reply:
Maybe, but it doesn't check whether a value is set, just whether the key is present ... although I guess it doesn't really matter: All real forms are actually going to have all of this of this metadata, and it will always have values.